### PR TITLE
fix #79121 61saat.com

### LINKED ISSUE #79121: www.61saat.com
### Issue URL (Ads)
 [https://www.61saat.com/spor/bursasporun-elektrikleri-yine-kesildi-h875085.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.61saat.com%2Fspor%2Fbursasporun-elektrikleri-yine-kesildi-h875085.html)
### Comment
Username: @Erengit
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#79121

### CODE REVIEW

**Comment by Sergey-Lyapin** [comment]
I can't reproduce, @wout-codes do you still see it?

**Comment by wout-codes** [comment]
Yes.

**Comment by Sergey-Lyapin** [comment]
@AdguardTeam/filters-maintainers please take a look

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -42,6 +42,9 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
 sgkbilgisi.com##.fdsphvn
+61saat.com#$##modal-ads { display: none !important; }
+61saat.com#$#.modal-backdrop { display: none !important; }
+61saat.com#$#body { overflow: visible !important; padding-right: 0 !important; }
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:47]: `padding-right: 0 !important;` is not required

Comment (wout-codes) [reply TurkishFilter/sections/specific.txt:47]: @Alex-302 On my end, the padding changes.

Comment (Alex-302) [reply TurkishFilter/sections/specific.txt:47]: I see no difference. Do you see horyzontal scroll without it or what?

Comment (wout-codes) [reply TurkishFilter/sections/specific.txt:47]: Nope. It's just normal. @zloyden Can explain better. 
image

Comment (Alex-302) [reply TurkishFilter/sections/specific.txt:47]: Now I see)

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**